### PR TITLE
ci: add workflow dispatch for updates in prisma/prisma

### PR DESCRIPTION
**Pull request by pantharshit00**
This adds workflow dispatch to prisma/prisma whenever it releases a new version. 

Follow up to https://github.com/prisma/prisma/pull/7640

The publish script was modified a bit to output the input version whenever it releases a new `latest` tag and not other npm tags.

### CODE REVIEW

File: scripts/publish.ts [hunk 1]
```diff
@@ -65,6 +65,12 @@ async function main(dryRun = false) {
     `pnpm publish --no-git-checks --tag ${npmTag}`,
     dryRun,
   )
+
+  // Print out version for workflow dispatch if npmTag is latest
+  if (npmTag === 'latest') {
+    console.log(`Printing version for workflow dispatch: ${npmTag}`)
+    console.log(`::set-output name=new_prisma_version::${npmTag}`)
```
Comment (janpio) [inline scripts/publish.ts:72]: Please add a comment here what this does - I assume this puts it somehow into GH Actions workflow context?

File: .github/workflows/publish-engines.yml [hunk 2]
```diff
@@ -39,6 +40,15 @@ jobs:
           commit_user_email: prismabots@gmail.com
           commit_author: prisma-bot <prismabots@gmail.com>
 
+      - name: Workflow dispatch to prisma/prisma for version update
+        if: ${{ steps.publish_script.outputs.new_prisma_version }}
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: Update Engines Version
```
Comment (janpio) [inline .github/workflows/publish-engines.yml:47]: This is by workflow name?

**Review by janpio — APPROVED** [review]

File: .github/workflows/publish-engines.yml (hunk 2 above)
Comment (pantharshit00) [reply .github/workflows/publish-engines.yml:47]: Yes

File: scripts/publish.ts (hunk 1 above)
Comment (pantharshit00) [reply scripts/publish.ts:74]: Yep, it exactly does that. I added the comment

**Pull request merged**